byu-calendar-row/byu-calendar-row.scss

```diff
@@ -208,5 +208,5 @@ p, div, a {
 }
 
 h1, h2, h3, h4 {
-	font-family: "Sentinel A", "Sentinel B";
+	font-family: Domine, "Sentinel A", "Sentinel B";
```

Comment:
You also need to include a basic fallback - 
font-family: 'Domine', serif; is the example google gives. Let's add that after Sentinel lines.